### ISSUE TITLE
How do I upload projects using Scratchattach?

### ISSUE THREAD

**Issue by NumbersTada** [TimMcCool/scratchattach#152]
can you give me some help?

**Comment by TimMcCool** [comment]
You can use ````py
project = session.connect_project("PROJECTID")
project.set_json(json_data)
```
 to change the project JSON of a project.

**Comment by NumbersTada** [comment]
But how do I create a new project?

**Comment by TimMcCool** [comment]
You can't create new projects with scratchattach

**Comment by NumbersTada** [comment]
OK

**Comment by NumbersTada** [comment]
Also, is it possible to fetch all projects of an user at once?

**Comment by RC9910-Scratch** [comment]
`user.projects(limit=None, offset=0)` -> this returns all the shared projects of a user as `scratch3.Project` objects

**Comment by NumbersTada** [comment]
That only returns the first 20 projects

**Comment by gamecraft312** [comment]
> That worked for an already existing project, but how do I create a new project?

you can use:
import webbrowser 

webbrowser.open_new_tab(https://scratch.mit.edu/projects/editor/) #this will make the ID and generate a new project

**Comment by Leaderofepic134** [comment]
> You can't create new projects with scratchattach

will this ever happen?

**Comment by TimMcCool** [comment]
> > You can't create new projects with scratchattach
> 
> will this ever happen?

Maybe

**Comment by Waakul** [comment]
you can send a request to https://scratch.mit.edu/projects/editor/ . I'm not sure how to get its id tho. You'll have to a list of your project from an API and check the id of the latest one.

**Comment by Waakul** [comment]
you could get the i using bs4 in "https://scratch.mit.edu/mystuff/" with all your cookies i believe, the link to it is in the a tag of the top most project. you could parse the id out of it.

**Comment by TheCommCraft** [comment]
You need to do a post request to https://projects.scratch.mit.edu/ and get "content-name" from the response.

Example(not guarranteed to work):
```python
import requests

session = ... # Create your session here

response = requests.post("https://projects.scratch.mit.edu/", headers=session._headers)
project_id = int(response.json()["content-name"])
```

**Comment by namtacs** [comment]
Look and discuss api here https://github.com/towerofnix/scratch-api-unofficial-docs/issues/37

**Comment by TimMcCool** [comment]
This is implemented in scratchattach 2.0 (will be released next week).
#242
